src/components/project/query.helpers.ts

```diff
@@ -80,6 +80,18 @@ export const projectListFilter = (filter: ProjectFilters) => (query: Query) => {
     ]);
   }
 
+  if (filter.partnerId != null) {
+    query
+      .match([node('partner', 'Partner', { id: filter.partnerId })])
```

Comment:
This doesn't need to be split from the reference below, it can be represented as a single path. Note that when using in where clause path condition you can't add variables to scope, so `node('', 'Partner', ...)`